### PR TITLE
Post ID fixes

### PR DESCRIPTION
**Pull request by christinem**
This is the first step in fixing the Check My POSt bugs (probably the biggest) - this fixes the issue where the new CS graph introduced new IDs for the nodes with more than one course associated with them.

### CODE REVIEW

File: public/js/graph/create_data.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 var math = [
-    'Calc1', 'Lin1', 'Sta1', 'Sta2'
+    'MAT135136137157Calc1', 'MAT221223240Lin1', 'STA247255257Sta1', 'Sta248261'
 ];
 
 // All CSC Courses
-var allCourses = ['CSC108', 'CSC148', 'CSC165', 'CSC207', 'CSC236',
-                  'CSC209', 'CSC258', 'CSC263', 'CSC300', 'CSC301',
+var allCourses = ['CSC108', 'CSC148', 'CSC165240', 'CSC207', 'CSC236240',
+                  'CSC209', 'CSC258', 'CSC236265', 'CSC300', 'CSC301',
```
Comment (david-yz-liu) [inline public/js/graph/create_data.js]: Should be `CSC263265`

File: public/js/graph/create_data.js [hunk 2]
```diff
@@ -23,9 +23,9 @@ var areaNames = ['theory', 'core', 'se', 'systems', 'hci',
 
 // Required for specialist
 var reqs = [
-    'CSC108', 'CSC148', 'CSC165', 'CSC207',
-    'CSC209', 'CSC236', 'CSC258', 'CSC263',
-    'CSC369', 'CSC373', 'Calc1', 'Lin1', 'Sta1'
+    'CSC108', 'CSC148', 'CSC165240', 'CSC207',
+    'CSC209', 'CSC236240', 'CSC258', 'CSC236265',
```
Comment (david-yz-liu) [inline public/js/graph/create_data.js]: `CSC263265`

File: public/js/post/update_post.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
              'filledTextboxes400': 0, 'filledTextboxesExtra': 0, 'majCount': 0, 'reqs': ['CSC108',
-             'CSC148', 'CSC165', 'CSC207', 'CSC236', 'CSC258', 'CSC263', 'Sta1', 'Calc1'], 'textboxes300': 2,
+             'CSC148', 'CSC165240', 'CSC207', 'CSC236240', 'CSC258', 'CSC263265', 'STA247255257Sta1', 
+             'MAT135136137157Calc1'], 'textboxes300': 2,
              'textboxes400': 1, 'textboxesExtra': 3, 'categories': 13, 'creditCount': 0, 'name': 'major'};
 var minor = {'index300': 0, 'index400': 0, 'categoriesCompleted': 0, 'filledTextboxesExtra': 0,
-             'reqs': ['CSC108', 'CSC148', 'CSC165', 'CSC207', 'CSC236'], 'textboxesExtra': 3, 'categories': 6,
-             'creditCount': 0, 'additionalMin200': ['CSC209', 'CSC258', 'CSC263'], 'name': 'minor'};
+             'reqs': ['CSC108', 'CSC148', 'CSC165240', 'CSC207', 'CSC236240'], 'textboxesExtra': 3, 'categories': 6,
+             'creditCount': 0, 'additionalMin200': ['CSC209', 'CSC258', 'CSC263253'], 'name': 'minor'};
```
Comment (david-yz-liu) [inline public/js/post/update_post.js]: CSC263265

File: public/js/post/update_post.js [hunk 4]
```diff
@@ -59,16 +61,20 @@ function resetValues() {
...
+                  'CSC373'], 'textboxes300': 3, 'textboxes400': 3,
+                  'textboxesExtra': 4, 'categories': 17, 'creditCount': 0, 'name': 'specialist'};
+    var major = {'index300': 0, 'index400': 0, 'categoriesCompleted': 0, 'filledTextboxes300': 0,
+                 'filledTextboxes400': 0, 'filledTextboxesExtra': 0, 'majCount': 0, 'reqs': ['CSC108',
+                 'CSC148', 'CSC165240', 'CSC207', 'CSC236240', 'CSC258', 'CSC263265', 'STA247255257Sta1', 
+                 'MAT135136137157Calc1'], 'textboxes300': 2,
+                 'textboxes400': 1, 'textboxesExtra': 3, 'categories': 13, 'creditCount': 0, 'name': 'major'};
+    var minor = {'index300': 0, 'index400': 0, 'categoriesCompleted': 0, 'filledTextboxesExtra': 0,
+                 'reqs': ['CSC108', 'CSC148', 'CSC165240', 'CSC207', 'CSC236240'], 'textboxesExtra': 3, 'categories': 6,
+                 'creditCount': 0, 'additionalMin200': ['CSC209', 'CSC258', 'CSC263253'], 'name': 'minor'};
```
Comment (david-yz-liu) [inline public/js/post/update_post.js]: CSC263265

**Comment by christinem** [comment]
@david-yz-liu thanks for the catches, I did a find and replace all and I must have put in the wrong number and didn't catch it since I had them correct on update_post.js. I'll make a new PR fixing them.

**Pull request merged**